app/delivery/send_to_providers.py

```diff
@@ -379,10 +379,9 @@ def provider_to_use(
     do_not_use_pinpoint = (
         has_dedicated_number
         or sending_to_us_number
-        or current_app.config["AWS_PINPOINT_SC_POOL_ID"] is None
-        or (current_app.config["AWS_PINPOINT_DEFAULT_POOL_ID"] is None and not using_sc_pool_template)
+        or not current_app.config["AWS_PINPOINT_SC_POOL_ID"]
```

Comment:
This is my lack of experience with Python - so forgive me - just want to confirm my understanding. This will render as false if AWS_PINPOINT_SC_POOL_ID is an empty string in the config? I would have thought it would only do that if null (which I guess is what you thought originally too hence the "none" from before?)

Reply:
yeah exactly, this is the "pythonic" way to check for an empty string
```
>>> not ""
True
>>> not "hello"
False
>>> "" is None
False
```